### PR TITLE
ehealthcaresolutions Bid Adapter : initial release

### PR DESCRIPTION
**Pull request by pranavsheth**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter  
- [ ] Updated bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

File: modules/ehealthcaresolutionsBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  isBidRequestValid: (bParam) => {
+    return !!(bParam.params.placement_id);
+  },
+  // Make a server request from the list of BidRequests
+  buildRequests: (bidRequests, serverRequest) => {
+    // Get Requests based on media types
+    return getBannerRequest(bidRequests, serverRequest, ENDPOINT_URL);
+  },
+  // Unpack the response from the server into a list of bids.
+  interpretResponse: (bResponse, bRequest) => {
```
Comment (patmmccann) [inline modules/ehealthcaresolutionsBidAdapter.js:29]: Seems like an opportunity to import this as well?

**Comment by ChrisHuie** [comment]
Can you please add documentation for both of these adapters in our docs repo -> https://github.com/prebid/prebid.github.io

**Comment by pranavsheth** [comment]
> Can you please add documentation for both of these adapters in our docs repo -> https://github.com/prebid/prebid.github.io

Hi @ChrisHuie 
we already had PR for tapnative, it is merged. I have created new PR for ehealthcaresolutions  - https://github.com/prebid/prebid.github.io/pull/5686

**Comment by ChrisHuie** [comment]
@pranavsheth can you please pull in the master branch. This fork is a bit behind and doesn't include tapnative being merged in master

**Comment by pranavsheth** [comment]
> @pranavsheth can you please pull in the master branch. This fork is a bit behind and doesn't include tapnative being merged in master

Hi @ChrisHuie 
we created this fork for tapnative and tapnative code is merged in master. Now we have used the same fork for this new adapter. so code is already upto date.

**Comment by pranavsheth** [comment]
> > > @pranavsheth can you please pull in the master branch. This fork is a bit behind and doesn't include tapnative being merged in master
> > 
> > 
> > Hi @ChrisHuie we created this fork for tapnative and tapnative code is merged in master. Now we have used the same fork for this new adapter. so code is already upto date.
> 
> Yeah. Didn't realize one of these adapters is already in master because the fork is behind and says you are adding both adapters in this pr. Please pull in master to resolve this discrepancy.

HI @ChrisHuie 
as suggested we have pulled from master to resolve this discrepancy. pl check and merged to master if everything is fine.
thanks.

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**